docs/installation.rst

```diff
@@ -23,8 +23,14 @@ not be moved.  The following command will generate a new directory called
 
    virtualenv ~/girder_env
 
-Now you can run ``source ~/girder_env/bin/activate`` to *enter* the new
-virtual environment.  Inside the virtual environment you can use ``pip``,
+Enter the virtual environment: 
+
+.. code-block:: none
+
+   source ~/girder_env/bin/activate
```

Comment:
This is a good change -- while we're at it, I vote we change from `source` to `.` for shell portability.